Dockerfile

```diff
@@ -48,6 +39,8 @@ RUN apt-get update && \
     git \
     jq
 
+# NOTE: keep in sync with the version installed in the dynamic-sidecar
+ARG DOCKER_COMPOSE_VERSION="2.27.1"
```

Comment:
thanks for cleaning this up and updating it.

Reply:
:)